src/models/RoomModel.js

```diff
@@ -0,0 +1,15 @@
+import db from '../apiCalls';
+
+class RoomModel {
+  constructor() {
+    this.allRooms = this.getAllRooms();
```

Comment:
Nik gave our group feedback that constructor properties should not hold methods. Consider assigning this property within a method instead.